cms/djangoapps/contentstore/views/course.py

```diff
@@ -261,6 +262,9 @@ def course_handler(request, course_key_string=None):
             if course_key_string is None:
                 return redirect(reverse("home"))
             else:
+                # Revoke sudo privileges from a request explicitly
```

Comment:
@dan-f Sorry for that I thought may be we need to revoke sudo privileges on course load too, I'll remove that.